### PR TITLE
dillo 3.0.5 (new formula)

### PR DESCRIPTION
**Pull request by tsutsu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/dillo.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-ssl",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/dillo", "--version"
```
Comment (woodruffw) [inline Formula/dillo.rb]: Can we get a more detailed test than just `--version`?

File: Formula/dillo.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  desc "Graphical web browser known for its speed and small footprint"
+  homepage "http://www.dillo.org/"
+  url "http://www.dillo.org/download/dillo-3.0.5.tar.bz2"
+  sha256 "db1be16c1c5842ebe07b419aa7c6ef11a45603a75df2877f99635f4f8345148b"
+
+  depends_on "openssl"
+  depends_on "fltk"
+
+  # for errors about Fl_Font.H (fixed in HEAD)
+  patch :DATA
```
Comment (MikeMcQuaid) [inline Formula/dillo.rb]: Please link to the HEAD commit/PR as a comment, thanks.

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/dillo.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--enable-ssl",
+                          "--prefix=#{prefix}"
+    system "make", "install"
```
Comment (stephengroat) [inline Formula/dillo.rb]: what about adding in a `system "make", "test"`?

Comment (MikeMcQuaid) [inline Formula/dillo.rb]: Like elsewhere: if it passes locally for you on macOS and takes <1m then it seems OK.

**Comment by fxcoudert** [comment]
Closing for now, please reopen when the comments are addressed.

**Pull request closed without merging**